restaurantgame.js

```diff
@@ -0,0 +1,292 @@
+  else if (action === "go to" && placePersonItem === "work") {
+    console.log(roomLookup[placePersonItem].describe())
+    return play()
+  }
+  else if (action === "go to" &&  placePersonItem === "lobby") {
+    console.log(roomLookup[placePersonItem].describe())
+    return play()
+  }
+  else if (action === "go to" &&  placePersonItem === "dining room") {
+    console.log(roomLookup[placePersonItem].describe())
+    return play()
+  }
+  else if (action === "go to" &&  placePersonItem === "kitchen") {
+    console.log(roomLookup[placePersonItem].describe())
+    return play()
+  }
+  else if (action === "go to" &&  placePersonItem === "bar") {
+    console.log(roomLookup[placePersonItem].describe())
+    return play()
+  }
+  
+
+  else if (action === "take the" && placePersonItem === 'sign') {
+    console.log(itemTable[placePersonItem].take())
+    return play()
+  }
+  else if (action === "take the" && placePersonItem === 'menu') {
+    console.log(itemTable[placePersonItem].take())
+    return play()
+  }
+  else if (action === "drop the" && placePersonItem === "key" && player.Inventory.includes("key")) {
+    game[0]= "key"
+    player.Inventory.pop()
+    console.log("You have dropped the key")
+    return play()
+  }
+  else if (action === "drop the" && placePersonItem === "key") {
+    console.log("You don't have the key")
+    return play()
+  }
+  else if (action === "read the" && placePersonItem === 'sign') {
+    console.log(itemTable[placePersonItem].read())
+    return play()
+  }
+  else if (action === "read the" && placePersonItem === 'menu') {
+    console.log(itemTable[placePersonItem].read())
+    return play()
+  }
+  else if (action === "take the" && placePersonItem === 'key') {
+    console.log(itemTable[placePersonItem].take())
+    return play()
+  }
+  else if (action === "take the" && placePersonItem === 'tip') {
+    console.log(itemTable[placePersonItem].take())
+    return process.exit()
+  }
+  else if (action === "eat the" && placePersonItem === "fries") {
+    console.log(itemTable[placePersonItem].eat())
+    return play()
+  }
+  if (action === "drink the" && placePersonItem === 'fiddlehead') {
+    console.log(itemTable[placePersonItem].eat())
+    return play()
+  }
+  else if (action === "go on" && placePersonItem === "hike") {
+    console.log("You decide to skip your first day of work- YOU ARE FIRED!")
+    process.exit()
+  }
+  else if (action === "check my" && placePersonItem === "inventory") {
+    console.log("You are carrying: " + player.Inventory)
+    return play()
+  }
+  else if (action === "check my" && placePersonItem === "status") {
+    console.log(player.status)
+    return play()
+  }
+  else if (action === "check the" && placePersonItem === "game") {
+    console.log(game)
+    return play()
+  }
+  else if (action === "exit the" && placePersonItem === "game") {
+    process.exit()
+  }
+  else {
+    console.log("Sorry I do not recognize that!! Try again")
+    return play()
+  }
+}
+
```

Comment:
If you decided to refactor this, you could create a array of commands that function with certain items, this would allow you clean up some of you code and trim down some of these else if statements. And with this many else if statements, its important to add notes so show what things are doing(ie these 4 else if statements are checking for movement, actions, interactions, etc.)